foyer/forcefields/xml/oplsaa.xml

```diff
@@ -284,10 +284,10 @@
   <Type name="opls_276" class="opls_276" element="C" mass="12.011"/>
   <Type name="opls_277" class="C_2" element="C" mass="12.01100" def="[C;X3]([O;X1])H" desc="C in aldehyde" doi="10.1021/ja9621760"/>
   <Type name="opls_278" class="O_2" element="O" mass="15.9994"  def="[O;X1][C;%opls_277]" desc="O in aldehyde" doi="10.1021/ja9621760"/>
-  <Type name="opls_279" class="HC"  element="H" mass="1.00800"  def="H[C;X3][O;X1]" desc="H-alpha in aldehyde and formamide" overrides="opls_185, opls_144" doi="10.1021/ja9621760"/>
+  <Type name="opls_279" class="HC"  element="H" mass="1.00800"  def="H[C;X3][O;X1]" desc="H-alpha in aldehyde and formamide" overrides="opls_185, opls_144, opls_485" doi="10.1021/ja9621760"/>
```

Comment:
The alpha hydrogen is always bonded not to the central carbon, but to the carbon off the central carbon. I.e.
`[H][C]([C;X3]([O;X1])[H])`